src/commands/uwu-ify.ts

```diff
@@ -34,8 +34,9 @@ const uwuify: Command = {
 				binding_info.str +
 				`\n` +
 				`  `.repeat(binding_info.line_number) +
-				`(format "${"~a ".repeat(binding_info.line_number).trim()}"` +
+				`(format "${"~a ".repeat(binding_info.line_number).trim()}" ` +
```

Comment:
Do you need the space if you remove the trim?

Reply:
i want the trim before the `"` but the space after it so that it looks like `(format "~a ~a" uwu1 uwu2)`

Reply:
otherwise it would look like `(format "~a ~a "uwu1 uwu2)`?

Reply:
trU

Reply:
where are the unit test!